MessageBird/Objects/Voice/Call.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
```

Comment:
Can you please delete the unused using statements here?

Reply:
They are all being used actually